assets/lokomotive-kubernetes/bootkube/assets.tf

```diff
@@ -109,14 +111,18 @@ data "template_file" "kubelet" {
 
 # Populate kubelet chart values file named kubelet.yaml.
 resource "local_file" "kubelet" {
-  content  = data.template_file.kubelet.rendered
+  count = var.disable_self_hosted_kubelet == false ? 1 : 0
+
+  content  = data.template_file.kubelet[0].rendered
```

Comment:
Does this work if kubelet is disabled? because then there will be no index 0 at this resource. Can Terraform handle that nicely nowadays? I was usually using `join("", data.template_file.kubelet.*.rendered)` IIRC.

Reply:
It works because the code won't be executed. Since we have that if statement before:

```
count = var.disable_self_hosted_kubelet == false ? 1 : 0
```

Reply:
Ah yes, that make sense, I think (you never know in Terraform world) :+1: